pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1617,6 +1617,21 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 		domain.Spec.Devices.Controllers = append(domain.Spec.Devices.Controllers, scsiController)
 	}
 
+	if needsMorePCIEControlers(vmi) {
```

Comment:
It feels to me this code deserves its own commit we a detailed commit message explaining why it is needed.

Reply:
isn't that what commit

> api, converter: specify the number of PCIe ports in the VM

is about ?